src/interpolations.jl

```diff
@@ -703,3 +699,35 @@ function value(ip::CrouzeixRaviart{2,1}, i::Int, ξ::Vec{2})
     i == 3 && return 1.0 - 2*ξ_y
     throw(ArgumentError("no shape function $i for interpolation $ip"))
 end
+
+################################################################
+# Wrapper Interolation for structural elements (shells, beams) #
+################################################################
+"""
+
+"""
+struct ShellInterpolation{dim_p,dim_s,shape,order,I<:Interpolation} <: Interpolation{dim_s,shape,order} 
+    ip::I
+end
```

Comment:
readoing this, wouldn't something like `StructuralInterpolation` be more precise?